src/Eurofurence.App.Server.Web/Controllers/ArtistsAlleyController.cs

```diff
@@ -47,10 +95,10 @@ public async Task<ActionResult> PutTableRegistrationStatusAsync([EnsureNotNull][
             else if (state == TableRegistrationRecord.RegistrationStateEnum.Accepted)
             {
                 await _tableRegistrationService.ApproveByIdAsync(id, "API:" + User.Identity.Name);
-                if (record.ImageId is { } imageId) await _imageService.SetRestricted(imageId, false);
+                if (record.ImageId is {} imageId) await _imageService.SetRestricted(imageId, false);
             }
 
-            return NoContent();
+            return Ok();
```

Comment:
NoContent is the correct success response to the PUT request, since it was successful and no body with data will be provided by the server. Please revert. (: